tests/sles4sap/redirection_tests/ensa2_kill_sapinstance.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Test module performs ENSA2 B<'Kill sapinstance'> test scenario.
+
+use parent 'sles4sap::sap_deployment_automation_framework::basetest';
+
+use warnings;
+use strict;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils qw(script_retry);
+use hacluster;
+use sles4sap::sap_host_agent qw(saphostctrl_list_instances);
+use sles4sap::sapcontrol;
+use sles4sap::console_redirection;
+
+=head1 SYNOPSIS
+
+Test module performs ENSA2 B<'Kill sapinstance'> test scenario. SAP instance process (ERS or ASCS) B<*.sap*> is killed
+using C<kill -9> command.
+Test module is based on usage of console redirection. Check B<tests/sles4sap/redirection_tests/README.md> for details.
+
+Test module requires following structure containing information:
+    $run_args->{scenarios} = [
+        'GottaCatchEmAll' => {
```

Comment:
what is the `GottaCatchEmAll` ? Is sort of scenario name?

Reply:
Sorry.. that was a dumb example. Trying to be funny at wrong place. I'll replace it with valid one.

Reply:
my doubt was only on this key (that all in all seems just only a name) and the resource one. Looking at the file with loadtest makes it clear to me now

Reply:
But thanks pointing out that example. Looking back it was just confusing.